debian/runonce.install

```diff
@@ -1,3 +1,5 @@
 src/runonce.sh /usr/share/runonce/
 src/runonce-generator /usr/lib/systemd/system-generators/
-systemd/runonce@.service /usr/lib/systemd/system/
+src/include.sh /usr/libexec/runonce/
```

Comment:
Can we move this to /usr/share/runonce/ so it's in the same dir as runonce.sh ?

Reply:
Done